app/v1/controllers/requestParsers/validators/FromAndToValidator.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators
+
+import v1.models.request.FromAndToRawData
+import v1.controllers.requestParsers.validators.validations.{DateFormatValidation, DateRangeValidation, MissingParameterValidation, NinoValidation}
+import v1.models.errors._
+
+trait FromAndToValidator[T <: FromAndToRawData] extends Validator[T] {
```

Comment:
I like the implementation, i'm just not entirely sure about the name. I would suggest 'ListValidator' but the transactions endpoint isn't _technically_ a list endpoint. Just think it needs to be clear that we're validating the nino as well.

Reply:
How about something like `ValidatorWithDateRange`?